src/Compute/Compute/VirtualMachine/Operation/GetAzureVMBootDiagnosticsDataCommand.cs

```diff
@@ -88,6 +88,11 @@ public override void ExecuteCmdlet()
 
             ExecuteClientAction(() =>
             {
+                LocalPath = ResolveUserPath(LocalPath);
+                if (!LocalPath.EndsWith(Path.DirectorySeparatorChar.ToString()))
+                {
+                    LocalPath = LocalPath + Path.DirectorySeparatorChar;
```

Comment:
@maddieclayton we should do `Path.Combine(LocalPath, Path.DirectorySeparatorChar)` here